setup.py

```diff
@@ -67,13 +17,17 @@ def get_extensions(is_test):
 
     print("Found preCICE version " + pkgconfig.modversion('libprecice'))
 
+    compile_args = ["-std=c++17"]
+    link_args = []
+    include_dirs = [numpy.get_include()]
+    bindings_sources = ["cyprecice/cyprecice.pyx"]
     compile_args += pkgconfig.cflags('libprecice').split()
 
-    if not is_test:
+    if os.environ.get('TOX_ENV_NAME') is not None:
```

Comment:
We currently use TOX_ENV_NAME to detect a testing scenario.
Maybe an env like `PYPRECICE_MOCKED` would be a better fit to simplify testing outside of tox environments.